app/[locale]/institute/sections/page.tsx

```diff
@@ -1,9 +1,91 @@
-import { WorkInProgressStatus } from '~/components/status';
+import Link from 'next/link';
+import { Fragment } from 'react';
+import { IconType } from 'react-icons';
+import {
+  MdLibraryBooks,
+  MdOutlineAccountBalance,
+  MdOutlineAdminPanelSettings,
+  MdOutlineElectricalServices,
+  MdOutlineGroupWork,
+  MdOutlineHealthAndSafety,
+  MdOutlineRealEstateAgent,
+  MdOutlineSecurity,
+  MdOutlineSportsTennis,
+  MdOutlineStore,
+} from 'react-icons/md';
+import { BsHddNetworkFill } from 'react-icons/bs';
 
-export default function Sections({
+import Heading from '~/components/heading';
+import { cn } from '~/lib/utils';
+import { db } from '~/server/db';
+import { getTranslations } from '~/i18n/translations';
+
+export default async function Sections({
   params: { locale },
 }: {
   params: { locale: string };
 }) {
-  return <WorkInProgressStatus locale={locale} />;
+  const sections = await db.query.sections.findMany();
+
+  const SectionIcons: Record<string, IconType> = {
```

Comment:
The convention for variable names is lower camel case
```suggestion
  const sectionIcons: Record<string, IconType> = {
```